resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -3,60 +3,61 @@
 set -ex
 
 ################
-# Usage: controller.sh <os> <version>
+# Usage: controller.sh <os> <version> <basepath>
 # The controller script does all of the jar compilation
 # and setup needed to build packages for <os>
 # The os string should be one of el, sles, debian, ubuntu,
 # or fedora
 # The version should be the numerical OS version for el,
 # sles, and fedora and the codename for debian/ubuntu.
+# The resulting artifacts will be stored in <basepath>
 ################
 build_os=$1
 build_ver=$2
+basepath=$3
```

Comment:
Should we have a default so that folks aren't required to set basepath?

Reply:
oops, i merged already :X

Reply:
:woman_shrugging: I don't know that this script is ever going to be used manually, it just gets executed from rake tasks

Reply:
That was my understanding of this script

Reply:
can always fix it later if that usecase ends up happening